cmake/Packaging.cmake

```diff
@@ -139,6 +138,48 @@ ly_install(FILES ${_cmake_package_dest}
     COMPONENT ${CMAKE_INSTALL_DEFAULT_COMPONENT_NAME}
 )
 
+# Do inital check on workspace path in the event that LY_3RDPARTY_PATH is misconfigured
+file(REAL_PATH "${CMAKE_CURRENT_SOURCE_DIR}/.." _root_path)
+if (EXISTS "${_root_path}/3rdParty/packages")
+    set(CPACK_LY_3P_PACKAGE_DIRECTORY "${_root_path}/3rdParty/packages")
+else()
+    set(CPACK_LY_3P_PACKAGE_DIRECTORY ${LY_3RDPARTY_PATH}/packages)
+endif()
```

Comment:
you can use `${LY_PACKAGE_UNPACK_LOCATION}` instead, thats where the packages are unpacked

Reply:
I had some unexpected behavior in CPack last time I tried using `LY_PACKAGE_UNPACK_LOCATION`; it pulled from `_CPack_Packages\WIX\o3de_0.0.0.0\`. It could have been some sort of naming collision or a mistake on my part, but I'll make an issue if that's the former.

Reply:
thats weird, that is defined within this environment (cmake configuration). I would double check, we should not be overwriting that location. Note that the definition of that variable may contain a non-expanded variable, but we expand it in line 54 of `cmake\3rdPartyPackages.cmake` so at the point you read it, it should be exapnded.
If you read this from within cpack, it wont be there, we need to pass it to cpack by setting a variable that has the prefix `CPACK_`